tests/specs/api/conversations.spec.js

```diff
@@ -43,6 +45,23 @@ describe('Conversations API', () => {
         });
     });
 
+    describe('#uploadImage', () => {
+        it('should call http', () => {
```

Comment:
Blob is a browser specific thing and tests are running on node only. To achieve this we probably should run a second suite of tests on karma. I'll file a card to do that in the future.